Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -173,6 +173,10 @@ public override AView FocusSearch(AView focused, [GeneratedEnum] FocusSearchDire
 			if (tabIndexes == null)
 				return base.FocusSearch(focused, direction);
 
+			// use OS default--there's no need for us to keep going if there's one or fewer tab indexes!
+			if (tabIndexes.Count <= 1)
+				return base.FocusSearch(focused, direction);
+
 			int tabIndex = element.TabIndex;
 			AView control = null;
 			int attempt = 0;
```

Comment:
@samhouts looking at the below code, it looks like the discovery of the next tab stop in the do while loop will only happen once, as maxAttempts = 0, and attempt starts at 0 but will be 1 at this stage, am I missing something?

Reply:
Ah sorry missed the out in the extension